### PR TITLE
Added new library in redux section.

### PR DESCRIPTION
**Pull request by aromalanil**
Added  _Redux Saga Middleware Dart and Flutter_ library to Redux section. This was requested in #4543

### CODE REVIEW

File: src/docs/development/data-and-backend/state-mgmt/options.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ A state container approach familiar to many web developers.
 * [Animation Management with Redux and Flutter][],
   a video from DartConf 2018 [Accompanying article on Medium][]
 * [Flutter Redux package][]
+* [Redux Saga Middleware Dart and Flutter][]
```
Comment (bilal-uslu) [inline src/docs/development/data-and-backend/state-mgmt/options.md]: Can you please add "by Bilal Uslu" at the end of line like the following;

* [Redux Saga Middleware Dart and Flutter][], by Bilal Uslu

**Review by bilal-uslu — APPROVED** [review]
Thanks, looks great,

File: src/docs/development/data-and-backend/state-mgmt/options.md (hunk 1 above)
Comment (aromalanil) [reply src/docs/development/data-and-backend/state-mgmt/options.md]: I have added the author name to the new commit in this PR

**Comment by bilal-uslu** [comment]
Thanks @aromalanil, it looks great 🙂

**Review by bilal-uslu — APPROVED** [review]
it looks great 👍

**Review by bilal-uslu — APPROVED** [review]

File: src/docs/development/data-and-backend/state-mgmt/options.md [hunk 2]
```diff
@@ -119,6 +120,7 @@ A simplified reactive state management solution.
 [Flutter Architecture Samples]: https://fluttersamples.com/
 [Flutter: State Management with Mobx]: https://www.youtube.com/watch?v=p-MUBLOEkCs
 [Flutter Redux package]: {{site.pub-pkg}}/flutter_redux
+[Redux Saga Middleware Dart and Flutter]: https://pub.dev/packages/redux_saga
```
Comment (sfshaza2) [inline src/docs/development/data-and-backend/state-mgmt/options.md:123]: Can you use our macro? So:

```
[Redux Saga Middleware Dart and Flutter]: {{site.pub-pkg}}/redux_saga
```

Comment (aromalanil) [reply src/docs/development/data-and-backend/state-mgmt/options.md:123]: Sure !

**Comment by aromalanil** [comment]
@sfshaza2  I have made the changes

**Pull request merged**